pkg/bootstrap/bootstrap_config_test.go

```diff
@@ -80,13 +80,17 @@ func TestGolden(t *testing.T) {
 		},
 		{
 			base: "running",
-			labels: map[string]string{
-				"ISTIO_PROXY_SHA":     "istio-proxy:sha",
-				"INTERCEPTION_MODE":   "REDIRECT",
-				"ISTIO_PROXY_VERSION": "istio-proxy:version",
-				"ISTIO_VERSION":       "release-3.1",
-				"POD_NAME":            "svc-0-0-0-6944fb884d-4pgx8",
-				"istio-locality":      "region.zone.sub_zone",
+			envVars: map[string]string{
+				"ISTIO_META_ISTIO_PROXY_SHA":     "istio-proxy:sha",
+				"ISTIO_META_INTERCEPTION_MODE":   "REDIRECT",
+				"ISTIO_META_ISTIO_PROXY_VERSION": "istio-proxy:version",
+				"ISTIO_META_ISTIO_VERSION":       "release-3.1",
+				"ISTIO_META_POD_NAME":            "svc-0-0-0-6944fb884d-4pgx8",
+				"POD_NAME":                       "svc-0-0-0-6944fb884d-4pgx8",
+				"POD_NAMESPACE":                  "test",
+				"ISTIO_META_istio-locality":      "region.zone.sub_zone",
```

Comment:
I don’t think you can generally have a - in an env var.

Reply:
good point. refactored to put directly in ISTIO_METAJSON_LABELS (which preserves prior behavior).